--- /usr/share/pve-manager/ext4/pvemanagerlib.js.orig    2015-04-28 06:38:55.000000000 +0200
+++ /usr/share/pve-manager/ext4/pvemanagerlib.js    2015-06-19 17:50:05.856801125 +0200
@@ -485,7 +485,7 @@
         success: function(response, opts) {
         var data = response.result.data;

-        if (data.status !== 'Active') {
+        if (false) {
             Ext.Msg.show({
             title: gettext('No valid subscription'),
             icon: Ext.Msg.WARNING,
