--- /usr/share/pve-manager/js/pvemanagerlib.js.orig	2020-05-10 09:44:21.832501834 +0200
+++ /usr/share/pve-manager/js/pvemanagerlib.js	2020-05-10 09:44:47.832272641 +0200
@@ -446,7 +446,7 @@
 	'p': gettext('Premium')
     },
 
-    noSubKeyHtml: 'You do not have a valid subscription for this server. Please visit <a target="_blank" href="https://www.proxmox.com/products/proxmox-ve/subscription-service-plans">www.proxmox.com</a> to get a list of available options.',
+    noSubKeyHtml: 'This server is receiving updates from the Proxmox VE No-Subscription Repository. Please visit <a target="_blank" href="http://www.proxmox.com/products/proxmox-ve/subscription-service-plans">www.proxmox.com</a> to get a list of available options.',
 
     kvm_ostypes: {
 	'Linux': [
@@ -41090,7 +41090,7 @@
     pveGuidePath: '/pve-docs/index.html',
     onlineHelp: 'getting_help',
 
-    invalidHtml: '<h1>No valid subscription</h1>' + PVE.Utils.noSubKeyHtml,
+    invalidHtml: '<h1>Community Edition</h1>' + PVE.Utils.noSubKeyHtml,
 
     communityHtml: 'Please use the public community <a target="_blank" href="https://forum.proxmox.com">forum</a> for any questions.',
 
